src/org/exist/launcher/Launcher.java

```diff
@@ -97,17 +92,15 @@ public static void main(final String[] args) {
         SwingUtilities.invokeLater(() -> new Launcher(args));
     }
 
+    private ServiceManager serviceManager;
     private SystemTray tray = null;
     private TrayIcon trayIcon = null;
-    private Optional<WrappedService> runningAsService;
     private SplashScreen splash;
     private Optional<JettyStart> jetty = Optional.empty();
```

Comment:
It looks like `jetty` is read and written from different states. It needs some sort of synchronization mechanism, it might be possible to get away with `volatile` I haven't checked the semantics of all the read/write paths to it, otherwise and `AtomicReference` or `synchronized` is needed.